Source/santad/Santad.mm

```diff
@@ -321,8 +321,11 @@ void SantadMain(std::shared_ptr<EndpointSecurityAPI> esapi, std::shared_ptr<Logg
   // execution policy appropriately.
   [authorizer_client enable];
   [tamper_client enable];
-  // Start monitoring any watched items
-  watch_items->BeginPeriodicTask();
+  if (@available(macOS 13.0, *)) {
```

Comment:
nit: I feel like this would be better to check within `BeginPeriodicTask`

Reply:
I'd like to keep it here for now unless there is a strong argument against it. There's nothing about the `WatchItems` object that is macOS 13 specific and if we move the check into that object, it will mean the tests for that method will not work on older OSs.